package/kernel/linux/modules/hwmon.mk

```diff
@@ -93,6 +93,20 @@ endef
 $(eval $(call KernelPackage,hwmon-ina2xx))
 
 
+define KernelPackage/hwmon-it87
+  TITLE:=it87 monitoring support
```

Comment:
Minor nit: other modules use uppercase characters before the number, I would prefer the same here for consistency